### PR TITLE
prepare for PyPI release v0.2.0

### PR DESCRIPTION
**Pull request by joelberkeley-pio**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -49,7 +49,7 @@
 author = 'The Trieste Contributors'
 
 # The full version, including alpha/beta/rc tags
-release = 'v0.1.0'
+release = 'v0.2.0'
```
Comment (joelberkeley-pio) [inline docs/conf.py:52]: we're uploading to the already existent trieste pypi package. That has versions 0.1.0 and 0.1.1, so bumping to 0.2.0 clearly delineates this from the old trieste project. We may choose to delete the 0.1.0 and 0.1.1 versions

File: setup.py [hunk 2]
```diff
@@ -19,8 +19,8 @@
 
 setup(
     name="trieste",
-    version="0.1.0",
-    author="The Trieste authors",
+    version="0.2.0",
+    author="The Trieste contributors",
```
Comment (joelberkeley-pio) [inline setup.py:23]: authors -> contributors to align with license text

File: setup.py [hunk 3]
```diff
@@ -19,8 +19,8 @@
 
 setup(
     name="trieste",
-    version="0.1.0",
-    author="The Trieste authors",
+    version="0.2.0",
```
Comment (joelberkeley-pio) [inline setup.py:22]: we're uploading to the already existent trieste pypi package. That has versions 0.1.0 and 0.1.1, so bumping to 0.2.0 clearly delineates this from the previous trieste project. We may choose to delete the 0.1.0 and 0.1.1 versions

File: docs/conf.py (hunk 1 above)
Comment (joelberkeley-pio) [inline docs/conf.py:52]: we should move to a VERSION file so we don't forget to update both versions, but I'm not going to do that now

**Review by vpicheny — APPROVED** [review]

File: docs/conf.py [hunk 4]
```diff
@@ -49,7 +49,7 @@
 author = 'The Trieste Contributors'
 
 # The full version, including alpha/beta/rc tags
-release = 'v0.1.0'
+release = '0.2.0'
```
Comment (joelberkeley-pio) [inline docs/conf.py:52]: i think the `v` was a mistake before

**Pull request merged**